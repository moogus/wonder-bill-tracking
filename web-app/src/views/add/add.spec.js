import React from 'react';
import { shallow } from 'enzyme';

import AddView from './add';

const render = ({ createPayment = () => {} } = {}) => {
  const props = { createPayment };
  return shallow(<AddView {...props} />);
};

describe('AddView', () => {
  test('default render is as expected', () => expect(render()).toMatchSnapshot());
});
