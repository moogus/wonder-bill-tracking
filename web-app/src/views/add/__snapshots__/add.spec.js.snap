// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddView default render is as expected 1`] = `
<PaymentForm
  buttonMessage="Add new payment"
  description="Keep track of your household spending by adding your bills"
  onClick={[Function]}
  title="Enter your details"
/>
`;
